test/unit/gateways/mundipagg_test.rb

```diff
@@ -136,6 +191,20 @@ def test_successful_authorize_with_partially_missing_address
     assert response.test?
   end
 
+  def test_successful_authorize_with_submerchant
+    options = @options.update(@submerchant_options)
+<<<<<<< HEAD
+=======
+
+>>>>>>> 069df08c78db676973551acac52ee979a80fe84a
```

Comment:
Some `git` cruff in here that needs to be tidied up.

Reply:
Removed!